outputs.tf

```diff
@@ -16,3 +16,8 @@ output "key_protect_name" {
   value       = ibm_resource_instance.key_protect_instance.name
   description = "Name of the Key Protect instance"
 }
+
```

Comment:
can we output metrics details too https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/kms_instance_policies#metrics

Reply:
Currently the `output "key_protect_instance_policies"` block is outputting all of the instance policies for the instance, would it be preferable to break these out into their own separate outputs?

Reply:
ah ok - we can leave it then